benchmarks/bench_quantize_vs_empirical_sol.py

```
@@ -0,0 +1,595 @@
+                current += 1
+                try:
+                    row = run_case(
+                        args=args,
+                        quantize=args.quantize,
+                        m=m,
+                        k=k,
+                        dtype=dtype,
+                        layout=layout,
+                        esol_args=esol_args,
+                    )
+                except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)